README.md

```diff
@@ -14,6 +14,8 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 ## Contents
 
+- [Awesome Actions ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-actions-awesomehttpsgithubcomsindresorhusawesome)
```

Comment:
@joelwmale can I ask why this was added?

Reply:
I don't actually know how this got added. I think my vscode extension automatically added this. I will revert this change sorry.

Reply:
@joelwmale no need to apologise, just wanted to check what was going on 😄